AGGREGATE.md

```diff
@@ -326,9 +329,11 @@ future versions of the API), the endpoint URL should also change.
 **Note:** The browser may need some mechanism to ensure that the same set of
 keys are delivered to different users.
 
-#### Optional: extended debugging reports
+### Optional: transitional debugging reports
 
-If [debugging](https://github.com/WICG/conversion-measurement-api/blob/main/EVENT.md#optional-extended-debugging-reports)
+#### Attribution-succeed debugging reports
```

Comment:
IMO the name should be either `Attribution-succeeded` or `Attribution-success`. `Attribution-succeed` doesn't seem right grammatically.

Reply:
Thanks, updated to Attribution-success debugging reports.